recipes/check-internet-speed/meta.yaml

```diff
@@ -10,18 +10,16 @@ source:
   sha256: 8be5c80f0d1f29e07529479cc40b3f2936196cd21be74d31495a818f981af917
 
 build:
-  #skip: False  # [win]
-  #noarch: python 
```

Comment:
Please don't remove the noarch.

Reply:
if i added noarc, i got this error...


❌ Non noarch packages should have python requirement without any version constraints.
❌ Non noarch packages should have python requirement without any version constraints.